geoips/interfaces/base.py

```diff
@@ -388,66 +387,14 @@ def _plugin_yaml_to_obj(cls, name, yaml_plugin, obj_attrs={}):
         plugin_type = f"{plugin_interface_name}Plugin"
 
         plugin_base_class = BaseYamlPlugin
+        # from IPython import embed as shell
+
+        # shell()
         if hasattr(cls, "plugin_class") and cls.plugin_class:
             plugin_base_class = cls.plugin_class
 
         return type(plugin_type, (plugin_base_class,), obj_attrs)(yaml_plugin)
 
-    def _create_unvalidated_plugins_cache(self, yaml_plugins):
-        """Create a cache of unvalidated plugin yamls.
-
-        These will be validated when they are actually used.
-        """
-        cache = {}
-        # If this is a list, split out all of the subs and store them all
-        # If this is any other family, just store it
-        for yaml_plg in yaml_plugins[self.name]:
-            if yaml_plg["family"] == "list":
-                try:
-                    yaml_plg = self.validator.validate(yaml_plg)
-                except ValidationError as resp:
-                    LOG.warning(
-                        f"{resp}: from plugin '{yaml_plg.get('name')}',"
-                        f"\nin package '{yaml_plg.get('package')}',"
-                        f"\nlocated at '{yaml_plg.get('abspath')}' "
-                    )
-                    # raise ValidationError(
-                    #     f"{resp}: from plugin '{yaml_plg.get('name')}',"
-                    #     f"\nin package '{yaml_plg.get('package')}',"
-                    #     f"\nlocated at '{yaml_plg.get('abspath')}' "
-                    # ) from resp
-                plg_list = self._plugin_yaml_to_obj(yaml_plg["name"], yaml_plg)
-                yaml_subplgs = {}
-                for yaml_subplg in plg_list["spec"][self.name]:
-                    try:
-                        subplg_names = self._create_plugin_cache_names(yaml_subplg)
-                        for subplg_name in subplg_names:
-                            yaml_subplgs[subplg_name] = deepcopy(yaml_subplg)
-                            yaml_subplgs[subplg_name]["interface"] = self.name
-                            yaml_subplgs[subplg_name]["package"] = yaml_plg["package"]
-                            yaml_subplgs[subplg_name]["relpath"] = yaml_plg["relpath"]
-                            yaml_subplgs[subplg_name]["abspath"] = yaml_plg["abspath"]
-                    except KeyError as resp:
-                        LOG.warning(
-                            f"{resp}: from plugin '{yaml_plg.get('name')}',"
-                            f"\nin package '{yaml_plg.get('package')}',"
-                            f"\nlocated at '{yaml_plg.get('abspath')}' "
-                            f"\nMismatched schema and YAML?"
-                        )
-                cache.update(yaml_subplgs)
-            else:
-                cache[yaml_plg["name"]] = yaml_plg
-        return cache
-
-    @staticmethod
-    def _create_plugin_cache_name(yaml_plugin):
-        """Create a plugin name for cache storage.
-
```

Comment:
@evrose54 @jsolbrig I Removed unused methods.

Reply:
See: https://github.com/NRLMMD-GEOIPS/geoips/pull/380/files

Reply:
This has been merged. Resolving.